src/test/routes/mocks/mockPostcodeInfoClient.ts

```diff
@@ -0,0 +1,16 @@
+import * as mock from 'mock-require'
+
+mock('@hmcts/postcodeinfo-client', './mockPostcodeInfoClient')
+
+export class PostcodeInfoClient {
+  constructor (public readonly apikey: string, public readonly request?, apiLocation?) {
```

Comment:
I think these properties can be private?

Reply:
typescript wouldn't compile if I had them private and didn't use them :(